terraform/samples/README.md

```diff
@@ -0,0 +1,7 @@
+# Mantl Sample Terraform Configuration Files
+
+If you are just starting out using Mantl and terraform, here are some
+sample terraform configuration files that work with the other files
+that we've written here. All you need to do is `cp` the sample that
+matches with your provider to the root of the mantl directory, fill in
+your credentials, and you're all set!
```

Comment:
We should link to the actual documentation on these things (the getting started guide)